packages/inotify/inotify.2.4.1/opam

```diff
@@ -0,0 +1,45 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Inotify bindings for OCaml"
+description: "Inotify bindings for OCaml"
+maintainer: ["whitequark <whitequark@whitequark.org>"]
+authors: ["whitequark <whitequark@whitequark.org>"]
+license: "LGPL-2.1"
+homepage: "https://github.com/whitequark/ocaml-inotify"
+doc: "https://whitequark.github.io/ocaml-inotify"
+bug-reports: "https://github.com/whitequark/ocaml-inotify/issues"
+depends: [
+  "dune" {>= "2.9"}
+  "base-unix"
+  "base-bytes"
+  "fileutils" {with-test & >= "0.4.4"}
+  "ounit2" {with-test & >= "2.0"}
+  "lwt" {with-test}
+  "ocaml" {>= "4.03"}
+  "odoc" {with-doc}
+]
+depopts: ["lwt"]
```

Comment:
I belive inotify is only available on linux (at least based on the failures on macOS)
```suggestion
depopts: ["lwt"]
available: os = "linux"
```

Reply:
how do I write that in a `dune-project` file?

Reply:
do you have to keep dune generating the opam file? There is probably a way with a template file but I'd highly recommend just stopping dune from autogenerating it.

Reply:
well that's disappointing :/
cc @rgrinberg would that be something we can add to dune? maybe arbitrary raw fields to be copied as-is into the opam file?

Reply:
Ah, I just tried a template with _only_ that field, and it works. Excellent, thank you :100: